tools/aequatus/aequatus-generator.py

```diff
@@ -35,15 +53,58 @@ def gene_to_json(fname):
 
 
 def join_json(tree_dict, gene_dict, cigar_dict):
+    global version
+
     aequatus_dict = {'tree': tree_dict,
                      'member': gene_dict}
 
     if cigar_dict:
         aequatus_dict['cigar'] = cigar_dict
+
+    aequatus_dict['ref'] = refGene
+    aequatus_dict['protein_id'] = refProtein
+    aequatus_dict['transcript_id'] = refTrancript
+    aequatus_dict['version'] = version
+    
     return aequatus_dict
 
+def parseGeneJSON(obj, id):
+    global refProtein
```

Comment:
`refProtein` is not modified in this function, do you mean `refTrancript`?

P.S.: global variables are considered evil in programming. All mutable objects (lists, dictionaries, class objects,...) are passed by reference in Python, so you can just add all these variables as function parameters.